Sources/PointFree/Account/Cancel.swift

```diff
@@ -221,7 +221,7 @@ let cancelEmailView = simpleEmailLayout(cancelEmailBodyView) <<< { owner, subscr
     Your \(subscription.plan.nickname) subscription has been canceled and will remain active through
     \(dateFormatter.string(from: subscription.currentPeriodEnd)).
     """,
-    template: .default,
+    template: .default(),
```

Comment:
The `.default` email template now takes a `Bool` parameter to determine if we want to show the PF logo or not. It seems kind of heavy handed in some types of emails, such as the one this PR adds. In fact, we may want to audit our existing emails and decide to omit the logo on some of them.